python/lsst/ap/association/diaForcedSource.py

```diff
@@ -237,9 +239,6 @@ def _calibrate_and_merge(self,
         output_catalog["time_processed"] = DateTime.now().toPython()
         # TODO: propagate actual flags (DM-42355)
 
-        # Drop superfluous columns from output DataFrame.
-        output_catalog.drop(columns=self.config.dropColumns, inplace=True)
```

Comment:
If you don't drop the columns here, it look like you may wind up with `x` and `y` after all, since they are renamed from `slot_Centroid_x` and `slot_Centroid_y` in this method.

Reply:
Note that I still drop the columns, I just moved it out of this method so that `x` and `y` are still available for the check in `_trim_to_exposure`

Reply:
Ah OK - so you drop them later in the process, not earlier - sounds good